### ISSUE TITLE
Scrobbling to last.fm

### ISSUE THREAD

**Issue by AreaNeofob** [themagiulio/bandcamp-desktop#41]
It would be cool if bandcamp-desktop could scrobble to last.fm. I'm guessing that the [Last.fm Music Discovery API](https://www.last.fm/api) is usable for this.

**Comment by sowinghustle** [comment]
Yes, please!!! I'd love to be able to use this desktop version instead of having to listen to music directly on Bandcamp's website, but not being able to use Last.fm alongside it is a major downside for me, hope the dev reads your request.

**Comment by themagiulio** [comment]
Hi, thanks for the suggestion.
I'll try to implement this feature in the future.
